src/main/java/seedu/address/model/module/ModuleCode.java

```diff
@@ -53,4 +53,7 @@ public String toString() {
         return value;
     }
 
+    public ModuleCode getClone() { // returns a defensive copy
+        return new ModuleCode(value);
+    }
```

Comment:
`value` is final, and is the only valuable attribute in this code so no need to go defensive

Reply:
`getClone()` may be useful in general as we extend our `ModuleCode` instance. Right now it's only a wrapper for a `String`, so the benefits are not clear yet.